backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -50,18 +50,31 @@ object BackendSettings extends BackendSettingsApi {
     // Validate if all types are supported.
     def validateTypes: Boolean = {
       // Collect unsupported types.
-      fields.map(_.dataType).collect {
-        case _: ArrayType =>
-        case mapType: MapType if mapType.keyType.isInstanceOf[StructType] =>
+      val unsupportedDataTypes = fields.map(_.dataType).collect {
+        case _: ByteType => "ByteType"
+        case _: ArrayType => "ArrayType"
+        case mapType: MapType if mapType.keyType.isInstanceOf[StructType] => "StructType as Key in MapType"
         // Parquet scan of nested map with struct as key type is not supported in Velox.
-        case _: ByteType =>
-      }.isEmpty
+      }
+      for (unsupportedDataType <- unsupportedDataTypes) {
+        // scalastyle:off println
+        println(
```

Comment:
It would be better to print with commonly used log.

Reply:
That will make this method unclear and I think it's okay to do print in this single case.